packages/sockethub/src/worker.ts

```diff
@@ -1,34 +1,25 @@
 import debug from 'debug';
-import domain from 'domain';
 import randToken from 'rand-token';
 import Store from 'secure-store-redis';
 import hash from 'object-hash';
+const { fork } = require('child_process');
 
 import crypto from './crypto';
 import services from './services';
 import SharedResources from './shared-resources';
 import config from './config';
+import {platform} from "os";
```

Comment:
Let's always use spaces between curly braces. Should probably be enforced by ESLint at some point soon.